roles/jupyter/defaults/main.yml

```
@@ -0,0 +1,9 @@
+---
+cull:
+  # Servers and notebooks that have been idle for longer then timeout (in seconds) will be culled.
+  timeout: '86400' # 1 day
+  # The maximum age (in seconds) of servers that should be culled even if they are active.
+  max_age: '604800' # 7 days
+jupyterhub_spawner_environment:
+  # Filesystem Spec configuration directory
+  FSSPEC_CONFIG_DIR: /etc/fsspec
```

Let's remove this one for now (also it would not work since the path is not accessible within the container):
```suggestion
jupyterhub_spawner_environment: {}
```